src/NotExistsClassException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+namespace JakubOnderka\PhpParallelLint;
```

Comment:
Suggestion to move exceptions to `Exception` namespace. Better overview of the project when browsing files. Easier onboarding for new developers to the project.

Reply:
This feedback belongs with PR #44.